src/index.ts

```diff
@@ -385,11 +392,13 @@ export interface Vendor {
   UsesNonCookieAccess?: boolean
```

Comment:
is this really the right casing for this?

Reply:
nope added correct and marked this as deprecated